### PR TITLE
Timestamp accuracy

### PR DESCRIPTION
**Pull request by ohadbitt**
#### Pull Request Description

Timestamp accuracy is not achieved using the current date formatter 
---

**Features:**
- Timestamp accuracy is not achieved using the current date formatter

### CODE REVIEW

File: connector/src/main/scala/com/microsoft/kusto/spark/datasink/RowCSVWriterUtils.scala [hunk 1]
```diff
@@ -35,25 +36,30 @@ object RowCSVWriterUtils {
       writer.writeStringField(json)
     }
   }
+  private def tss (l:Long, timeZone: ZoneId) ={
+    LocalDateTime.ofInstant(Instant.EPOCH.plus(l, ChronoUnit.MICROS), timeZone).toString
```
Comment (yihezkel) [inline connector/src/main/scala/com/microsoft/kusto/spark/datasink/RowCSVWriterUtils.scala:40]: I think this method would make more sense returning a LocalDateTime. The caller can call "toString" on it.
Plus, that would be consistent with the other "writer.writeStringField" calls below.

File: connector/src/test/scala/com/microsoft/kusto/spark/WriterTests.scala [hunk 2]
```diff
@@ -142,8 +139,8 @@ class WriterTests extends FlatSpec with Matchers {
         "asd2" -> Row(Array("stringVal2\b\f")))
     )
 
-    // 2018-06-18 16:04:00.0 in GMT
-    val ts = new java.sql.Timestamp(1529337840000l)
+    val tz  = ZonedDateTime.parse("2018-06-18T15:00:00.123456789+04:00")
```
Comment (yihezkel) [inline connector/src/test/scala/com/microsoft/kusto/spark/WriterTests.scala:142]: I could be missing it, but I don't see an assert that this or any non-UTC LocalDateTime went through the writer and came out the other end as expected.

File: connector/src/main/scala/com/microsoft/kusto/spark/datasink/RowCSVWriterUtils.scala [hunk 3]
```diff
@@ -35,25 +36,30 @@ object RowCSVWriterUtils {
       writer.writeStringField(json)
     }
   }
+  private def tss (l:Long, timeZone: ZoneId) ={
```
[the remark this answers is not in the document]
Comment (ohadbitt) [reply connector/src/main/scala/com/microsoft/kusto/spark/datasink/RowCSVWriterUtils.scala:39]: oo right - that was supposed to be temporary

File: connector/src/main/scala/com/microsoft/kusto/spark/datasink/RowCSVWriterUtils.scala (hunk 1 above)
Comment (ohadbitt) [reply connector/src/main/scala/com/microsoft/kusto/spark/datasink/RowCSVWriterUtils.scala:40]: thats right

File: connector/src/test/scala/com/microsoft/kusto/spark/WriterTests.scala (hunk 2 above)
Comment (ohadbitt) [reply connector/src/test/scala/com/microsoft/kusto/spark/WriterTests.scala:142]: line 282
it sohuld equal to 
sdf.format(ts.toInstant.atZone(tz.getZone))

**Review by yihezkel — APPROVED** [review]

**Pull request merged**